ts/components/ui/LogoPaymentHuge.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { Image } from "react-native";
+import paypalLogoImage from "../../../img/wallet/payment-methods/paypal-logo.png";
+// sadly no svg is available for paypal, since on Figma an image is used
+import BpayLogo from "../../../img/wallet/payment-methods/bpay_logo_full.svg";
+import { BankLogoOrSkeleton } from "./utils/baseComponents/BankLogoOrLoadingSkeleton";
+type LogoPaymentHugeProps = {
+  dimensions: { height: number; width: number };
+} & (
+  | {
+      icon: "payPal" | "bpay";
+    }
+  | {
+      abiCode: string;
+    }
+);
+
+export const LogoPaymentHuge = (props: LogoPaymentHugeProps) => {
```

Comment:
```suggestion
export const LogoPaymentExtended = (props: LogoPaymentHugeProps) => {
```
Sounds like a more appropriate name. What do you think? `Huge` refers to the size, but here we have an **extended** version of the logo, which can be the same size or even smaller than the standard one.

Reply:
addressed in [92989bc](https://github.com/pagopa/io-app/pull/4781/commits/92989bcc9bd38990fbb30e41b1b7d55406f2f8f2)